### PR TITLE
#122 logger file added and some loggs added successfully

### PR DESCRIPTION
**Pull request by SinghHimadri**
I am creating a draft PR please have a look and suggest changes. Specify the format of the logger and few must include instances I have added few to make sure I am working in right direction.

### CODE REVIEW

File: wplay/utils/logg.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import logging
+formatter = logging.Formatter('%(asctime)s %(levelname)s %(message)s')
+
+
+def setup_logger(name, log_file, level=logging.INFO):
+    """To setup as many loggers as you want"""
+
+    handler = logging.FileHandler(log_file)
```
Comment (rpotter12) [inline wplay/utils/logg.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5141068)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- wplay/utils/logg.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5141068&bid=16888780)

**Comment by SinghHimadri** [comment]
Is my work not correct?

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- wplay/utils/logg.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5141068&bid=16888780)

File: wplay/utils/logg.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import logging
+formatter = logging.Formatter('%(asctime)s %(levelname)s %(message)s')
+# endregion
+
+
+#region for logger class
+def setup_logger(name, log_file, level=logging.INFO):
+    """To setup as many loggers as you want"""
+
+    handler = logging.FileHandler(log_file)
```
Comment (rpotter12) [inline wplay/utils/logg.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5141068)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- wplay/utils/Logger.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5141068&bid=16888780)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- wplay/utils/Logger.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5141068&bid=16888780)

**Comment by xandao6** [comment]
Nice work, hope you finish soon.

**Comment by SinghHimadri** [comment]
Can I send the PR by evening? Please extend this for few hours.

**Pull request closed without merging**